src/node_buffer.cc

```diff
@@ -1223,6 +1223,18 @@ static void EncodeInto(const FunctionCallbackInfo<Value>& args) {
   results[1] = written;
 }
 
+static void IsUtf8(const FunctionCallbackInfo<Value>& args) {
+  Environment* env = Environment::GetCurrent(args);
+  CHECK_EQ(args.Length(), 1);
+  CHECK(args[0]->IsTypedArray() || args[0]->IsArrayBuffer());
```

Comment:
This is missing `IsSharedArrayBuffer()`?

Reply:
You're right. Adding it!